x/valset-pref/validator_set.go

```diff
@@ -29,6 +29,44 @@ func (k Keeper) SetupValidatorSetPreference(ctx sdk.Context, delegator string, p
 	return nil
 }
 
+// TODO MAYBE: Check if there are any banned assets and ways to handle them
```

Comment:
Mmm, idk if this TODO is worth leaving in the code base. Discussion should happen first with community before considering any implementation